pkg/pillar/cmd/upgradeconverter/movepatchenvelopes.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package upgradeconverter
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	fileutils "github.com/lf-edge/eve/pkg/pillar/utils/file"
+)
+
+const (
+	srcDir  = "/zedrouter/AppInstMetaData"
+	destDir = "/msrv/AppInstMetaData"
+)
+
+func exists(path string) (bool, error) {
```

Comment:
Don't we already have a function like this in some package?

Reply:
Yep, found it and fixed

Reply:
function exists can be now removed

Reply:
I removed my implementation and I'm using the one which is in utils, I still need to know if src and dst exist before copying

Reply:
Did you push the removal of your `exists`? I can still see it here.

Reply:
Didn't pull the trigger :D